Sources/ComposableArchitecture/Store.swift

```diff
@@ -412,10 +394,13 @@ public final class Store<State, Action> {
       // as part of state publisher updates,
       // process them now
       if !self.bufferedActions.isEmpty {
-        send(self.bufferedActions.removeLast(), file: file, line: line)
+        if let task = send(self.bufferedActions.removeLast(), file: file, line: line) {
+          tasks.wrappedValue.append(task)
+        }
```

Comment:
Oof. This whole function is honestly harder to reason about now. And this part alone tripped me up. :-\